guides/common/modules/ref_webhooks-templates.adoc

```diff
@@ -0,0 +1,12 @@
+[id="webhooks-templates_{context}"]
+= Templates
```

Comment:
This title is extremely vague. 
Mod docs style would require a descriptive title. Descriptive titles help with search engine optimisation. 
I think this document would be really benefit from Ingrid's minimalism session. @ryandeussing do you think you could help @ianf77 set this up for Ingrid's group to review this document? It would be a hugely beneficial learning experience.